### ISSUE TITLE
🐛 [Bug] Login design lack, have the Header items left and right aligned

### ISSUE THREAD

**Issue by Tirokk** [Human-Connection/Human-Connection#1246]
## :bug: Bugreport

On the login page the items in the Header should be left and right aligned with the login box.

### Steps to reproduce the behavior
1. go to the login page (may logout)

<img width="1812" alt="Bildschirmfoto 2019-08-09 um 17 49 34" src="https://user-images.githubusercontent.com/25344101/62870732-6c94c800-bd1a-11e9-946c-7aeef4415cc6.png">

**Comment by gha-gozu** [comment]
@Tirokk Do you mind if I'm involved in this one?

**Comment by mattwr18** [comment]
It would be great if you wanted to help with this @gha-gozu, have you gotten set up with the repo already?

**Comment by gha-gozu** [comment]
@mattwr18 Not yet, should I fork first? And no, I haven't joined the discord server.

**Comment by mattwr18** [comment]
No, please don't fork cause we have some encrypted variables in our travis setup and it will never pass the build from forks. Have you seen our docs [here](https://docs.human-connection.org/human-connection/installation)? 

There are two ways to get set up, either with `Docker` or set up everything locally. We can also have a pairing session if you want/need. 

Our discord server invite link is http://hc.world/discord
You can join us, and you'll keep up to date on all our regularly scheduled pairing sessions, and also be able to get ahold of us more quickly :smile_cat:

**Comment by gha-gozu** [comment]
I have seen it, only for the installation part. The discord invite there is expired.

I'll set up mine with docker, it's easier that way. Thank you for the consideration, but I'd like to try it on my own. If needed anything, I'll be sure to look for someone.

**Comment by Tirokk** [comment]
@gha-gozu I would love when your involved here! 💫

**Comment by gha-gozu** [comment]
@Tirokk How do I do that?

**Comment by Tirokk** [comment]
@gha-gozu at the top right click on assignees and chose yourself.

**Comment by gha-gozu** [comment]
@Tirokk Is it just me or is there nothing to click there?

**Comment by Tirokk** [comment]
Hmm …
I can assign you … ?!
I have done!

**Comment by gha-gozu** [comment]
Thank you!

**Comment by mattwr18** [comment]
I have added you to our volunteers team @gha-gozu 
now you should be able to push directly to you feature branch on the `Human-Connection` repo :smile:

**Comment by gha-gozu** [comment]
@mattwr18 Thanks!
